src/domain/swaps/entities/__tests__/order.builder.ts

```diff
@@ -0,0 +1,103 @@
+import { Order } from '@/domain/swaps/entities/order.entity';
+import { Builder, IBuilder } from '@/__tests__/builder';
+import { faker } from '@faker-js/faker';
+import { getAddress } from 'viem';
+
+export function orderBuilder(): IBuilder<Order> {
+  return new Builder<Order>()
+    .with('sellToken', getAddress(faker.finance.ethereumAddress()))
+    .with('buyToken', getAddress(faker.finance.ethereumAddress()))
+    .with(
+      'receiver',
+      faker.datatype.boolean()
+        ? getAddress(faker.finance.ethereumAddress())
+        : null,
+    )
+    .with('sellAmount', faker.number.bigInt({ min: 1 }))
+    .with('buyAmount', faker.number.bigInt({ min: 1 }))
+    .with('validTo', faker.date.future().getTime())
+    .with('appData', faker.string.hexadecimal())
```

Comment:
Out of scope of this PR, but is there a reason we aren't using the `HexSchema` for the `faker.string.hexadecimal` values here?

Reply:
`appData` shouldn't be considered an hexadecimal but a stringified JSON. I will update it.

Reply:
Changed in [e3a4a9c](https://github.com/safe-global/safe-client-gateway/pull/1299/commits/e3a4a9c33aac2d31572dd3cb51e6cacec9ed3515)